styleguide/source/assets/scss/05-pages/_article.scss

```diff
@@ -34,3 +34,13 @@
   @include gutter($margin-bottom-full...);
   background-color: $white;
 }
+
+.joe__page-content--article {
+  padding-top: 5px;
+}
+
+sup {
```

Comment:
The styling needs to move to the `styleguide/source/assets/scss/00-base/_typography.scss` file for `sup` element. PR #10 Has a similar approach.

Reply:
We could also do something like this, for `sup` element: 
```
    line-height: 1;
    font-size: 80%;
```